common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -319,6 +322,107 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
         return true;
     }
 
+    public boolean containsKey(Object key) {
```

Comment:
@nmittler can we do something like `containsKey(@primitiveType@)` i.e. `containsKey(int key)`? same for all the other methods.